### DISCUSSION TITLE
N + 1 problem seems to persist

### DISCUSSION THREAD

**Discussion by omerman** [prisma/prisma#5519] [Q&A]
I wanted to check with you guys regarding n+1 issue & DataLoader mechanism.

I opened a repo https://github.com/omerman/prisma-type-graphql-n1 that has typegraphql & prisma installed.
All you need to do is modify your .env to match a postgres db, and hit yarn dev.

When adding multiple Posts & Comments (two Posts and a single Comment per Post will be enough) and running this basic query ->

```graphql
  posts {
    id
    comments {
      id
      info
    }
  }
```
You'll quickly notice that the number of queries logged grow for each Post added. I believe this should not happen.
I dont understand why it does. I'd expect the batching mechanism of prisma to kick in and cure the number of queries ;)

Help?

**Comment by nikolasburk** [comment]
Hey @omerman, thank you so much for raising this and for providing a reproduction! Do you mind [reposting this discussion as an actual issue](https://github.com/prisma/prisma/issues/new), that way it'll get picked up by our Engineering team and they will look into the problem! 🙂

**Reply by omerman** [reply]
Done :) #5528 - I even tested it using an existing prisma example to double check that there is an issue.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
